HLTrigger/NGTScouting/python/hltTriggerAcceptFilter_cfi.py

```diff
@@ -0,0 +1,20 @@
+import FWCore.ParameterSet.Config as cms
+from HLTrigger.HLTfilters.triggerResultsFilter_cfi import triggerResultsFilter as _triggerResultsFilter
+
+hltTriggerAcceptFilter = _triggerResultsFilter.clone(
+    usePathStatus = cms.bool( True ),
+    hltResults = cms.InputTag( "" ),
+    l1tResults = cms.InputTag( "" ),
```

Comment:
This filter still checks the phase-1 uGT trigger bits and not the P2GT one: see [here](https://github.com/cms-sw/cmssw/blob/master/HLTrigger/HLTfilters/plugins/TriggerResultsFilter.cc#L89-L91)

Reply:
I don't think it matters. 

> Leave empty to ignore the L1T results.

The filter is purely technical and just checks that "some" HLT path did run.